acceptance-tests/src/main/java/io/blueocean/ath/SmartWebElement.java

```diff
@@ -59,8 +81,15 @@ protected WebDriver getDriver() {
      * Gets the first matching element
      * @return see description
      */
-    public WebElement getElement() {
-        return getElements().iterator().next();
+    public WebElement getElement() throws NoSuchElementException {
+        List<WebElement> elements = getElements();
+        if (elements == null || elements.isEmpty()) {
+            throw new NoSuchElementException("Nothing matched to click on for: " + expr);
```

Comment:
The reference to "click" here might be copy/paste error?

Reply:
ah yes, will fix; oops